README.rst

```diff
@@ -57,6 +57,20 @@ Highlights
 
 ----
 
+DALI Roadmap
+------------
+
+|dali-roadmap-link|_ a high-level overview of our 2021 plan. You should be aware that this
+roadmap may change at any time and the order below does not reflect any type of priority.
+
+We strongly encourage you to comment on our roadmap and provide us feedback on the mentioned
+GitHub issue.
+
+... |dali-roadmap-link| replace:: The following represents
```

Comment:
looks like too many dots, something is broken: https://github.com/JanuszL/DALI/blob/add_roadmap/README.rst

Reply:
Fixed